syncs/github-discussions/query.gql

```diff
@@ -0,0 +1,52 @@
+query($owner: String!, $repo: String!, $perPage: Int! $cursor: String) {
+  repository(owner: $owner, name: $repo) {
+    discussions(first: $perPage, after: $cursor) {
+      nodes {
+        id
+        activeLockReason
+        isAnswered
+        answer { id }
+        answerChosenAt
+        answerChosenBy { login }
+        author { login }
+        authorAssociation
+        body
+        category { createdAt description emoji id isAnswerable name slug updatedAt }
+        comments(first: 100) {
+          totalCount
+          nodes {
+            id
+            author { login }
+            authorAssociation
+            body
+            createdAt
+            deletedAt
+            isMinimized
+            minimizedReason
+            reactions { totalCount }
+            replyTo { id }
+            updatedAt
+            upvoteCount
+            url
+          }
+        }
```

Comment:
It looks like `comments` only returns the "parent" comments (i.e. `replyTo.id` is always `null`), so to get the `replies` we need to request them for each `comment`, and because we must not exceed the maximum of 500'000 potential nodes, we can request "only" 50 comments and 50 replies. This should suffice for most cases.

```suggestion
        comments(first: 50) {
          totalCount
          nodes {
            id
            author { login }
            authorAssociation
            body
            createdAt
            deletedAt
            isMinimized
            minimizedReason
            reactions { totalCount }
            replies(first: 90) {
              totalCount
              nodes {
                id
                author { login }
                authorAssociation
                body
                createdAt
                deletedAt
                isMinimized
                minimizedReason
                reactions { totalCount }
                replyTo { id }
                updatedAt
                upvoteCount
                url
              }
            }
            replyTo { id }
            updatedAt
            upvoteCount
            url
          }
        }
```